embark

```diff
@@ -0,0 +1,37 @@
+set notification-error-bg           "#1E1C31"
+set notification-error-fg           "#F48FB1"
+set notification-warning-bg         "#1E1c31"
+set notification-warning-fg         "#FFE6B3"
+set notification-bg                 "#1E1C31"
+set notification-fg                 "#CBE3E7"
+
+set completion-bg                   "#1E1C31"
+set completion-fg                   "#CBE3E7"
+set completion-group-bg             "#87DFEB"
+set completion-group-fg             "#2D2B40"
+set completion-highlight-bg         "#FFE6B3"
+set completion-highlight-fg         "#2D2B40"
+
+set index-bg                        "#1E1C31"
+set index-fg                        "#CBE3E7"
+set index-active-bg                 "#8A889D"
+set index-active-fg                 "#2D2B40"
```

Comment:
This should be able to stay as the original foreground color but double check after changing the background color.

Reply:
Let me check this...